store/dor-variables.js

```diff
@@ -0,0 +1,206 @@
+      region:
+        'CO₂ absorbed from the atmosphere compared to the removal potential induced by extracting CO₂. This metric accounts for both ocean uptake of atmospheric CO₂ after the DOR intervention and any losses from storage of the extracted CO₂. Higher values indicate more effective carbon removal.',
+    },
+    threshold: 0.001,
+    overview: true,
+    url: 'https://carbonplan-dor-efficiency.s3.amazonaws.com/v3/store1b.zarr',
+    variables: [
+      {
+        variable: 'DOR_efficiency',
+        colorLimits: [0, 1],
+        colormap: 'cool',
+        unit: 'CO₂ absorbed / potential removal',
+        graphLabel: 'Net efficiency',
+        graphUnit: '',
+      },
+    ],
+  },
+
+  // FG_CO2: {
+  //   label: 'Spread of CO₂ uptake',
+  //   description: {
+  //     overview:
+  //       'Percentage of cumulative CO₂ uptake taking place within the specified distance from the center of the intervention region. Select a region to view additional experimental outputs.',
+  //     region:
+  //       'Percentage of cumulative CO₂ uptake taking place within the specified distance from the center of the intervention region.',
+  //   },
+  //   threshold: 0.001,
+  //   overview: true,
+  //   url: 'https://carbonplan-dor-efficiency.s3.amazonaws.com/v3/cumulative_FG_CO2_percent.zarr',
+  //   optionsTooltip:
+  //     'View the percentage of cumulative CO₂ re-uptake taking place within 500 km, 1000 km, or 2000 km of the intervention center.',
+  //   variables: [
+  //     {
+  //       variable: 'FG_CO2_percent_cumulative',
+  //       colorLimits: [0, 100],
+  //       colormap: 'cool',
+  //       optionIndex: 0,
+  //       label: '500 km',
+  //       graphLabel: 'Uptake percentage',
+  //       unit: '%',
+  //       graphUnit: '',
+  //     },
+  //     {
+  //       variable: 'FG_CO2_percent_cumulative',
+  //       colorLimits: [0, 100],
+  //       colormap: 'cool',
+  //       optionIndex: 1,
+  //       label: '1000 km',
+  //       graphLabel: 'Uptake percentage',
+  //       unit: '%',
+  //       graphUnit: '',
+  //     },
+  //     {
+  //       variable: 'FG_CO2_percent_cumulative',
+  //       colorLimits: [0, 100],
+  //       colormap: 'cool',
+  //       optionIndex: 2,
+  //       label: '2000 km',
+  //       graphLabel: 'Uptake percentage',
+  //       unit: '%',
+  //       graphUnit: '',
+  //     },
+  //   ],
+  // },
+  DIC_SURF: {
+    label: 'Surface dissolved inorganic carbon',
+    description: {
+      region:
+        'Extracting CO₂ from the surface ocean creates a dissolved inorganic carbon (DIC) deficit. A larger deficit means more potential for the ocean to absorb CO₂ from the atmosphere.',
+    },
+    threshold: -0.00001,
+    optionsTooltip:
+      'View the change in DIC. Total values are not available for this variable.',
```

Comment:
Hmmm it seems like we should actually be able to show the `total` values for this variable. If I recall correctly, the only reason we decided to hide the total value for the integrated DIC variable was to avoid the confusion of showing bathymetry patterns, which wouldn't affect the surface variable. It also looks like the total `band` is present in `store2.zarr`! What do you think?